RELEASE-NOTES.rst

```diff
@@ -18,6 +18,221 @@
+   then the ``issue`` field is required, to describe the issuance
+   details and issue status. Issue items can only be attached to
+   holdings of serial type.
+   The field ``item_type`` that indicates the circulation category will
+   be removed later. The item circulation status is described in the
+   ``item.status`` field, while the issue status is described in the
+   ``item.issue.status``.
+-  Adds ``notes`` field in the item JSON schema. Four types of notes have been
+   added, two regarding their audience (public or staff) and two regarding a
+   circulation operation (checkin/checkout). The notes are displayed according
+   to their type: public notes are publicly displayed on the document detail
+   view of the public interface, staff notes are displayed on the same view but
+   in the professional interface, checkin notes are displayed as a permanent
+   alert as the item is checked in, and checkout notes are displayed as a
+   circulation transaction occurs.
+-  Uses ``JSONSCHEMAS_REPLACE_REFS = True`` to resolve JSON reference
+   before serving the schema.
+-  Removes the ``document-minimal-v0.0.1.json`` schema, as it is not
+   used.
+
+Acquisition
+-----------
+
+-  Receive an issue:
+
+   -  Allows the librarian to receive new issues through the holdings
+      record. The system, based on the holdings pattern, compute the
+      next issue pattern and expected date. The librarian is able to add
+      irregular or exceptional issues.
+   -  Updates automatically the ``next_expected_date`` after a
+      successful receive of a regular issue (``expected_date`` of the
+      receive issue plus the pattern frequency).
+
+API
+---
+
+-  Corrects the process used when starting a delayed bulk indexing
+   (switch from ``invenio-indexer`` to ``IlsRecordindexer``).
+-  Uses the standard JSON schema end point (``/schemas``).
+-  Restricts the receipt of issue to the librarians of the owning
+   library of the holdings record.
+-  The pattern preview API returns the ``issue_display_text`` (based on
+   the preview template) and the ``expected_date``.
+-  The holding API is able to receive the next regular issue.
+
+Documentation
+-------------
+
+-  Documents all circulation actions, trying to be the most
+   comprehensive in the context of a library network with complex
+   internal circulation workflow. Actions, scenarios and chart can be
+   found in
+   ```/doc/README.md#circulation`` <https://github.com/rero/rero-ils/blob/dev/doc/README.md#circulation>`__.
+   In the same move, the babel configuration has been cleaned.
+-  Removes unnecessary documentation in the ``LICENSE`` file.
+-  Fixes the ``AUTHORS.rst`` file (wrong indentation).
+-  Improves the github issue templates to add automatically different
+   label to issue, depending on the type of issue (bug, correction,
+   enhancement, etc.). This should also ease the process of creating and
+   triage issues.
```

Comment:
I think we cannot put a substantive in this place. It must be a verb (present participle).

```suggestion
   sorting issues.
```